### PR TITLE
modify client api to fit with server changes

### PR DESCRIPTION
**Pull request by zaclinsage**

### CODE REVIEW

File: bridge-api/paths/studies/v3_studies_init.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+post:
+    operationId: createStudyAndUser
```
Comment (alxdarksage) [inline bridge-api/paths/studies/v3_studies_init.yml]: createStudyWithUsers?

File: bridge-api/paths/studies/v3_studies_init.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    summary: Create a new study with given user list
```
Comment (alxdarksage) [inline bridge-api/paths/studies/v3_studies_init.yml]: , assigning the given Synapse users as admins of the team/project

File: bridge-api/paths/studies/v3_studies_self_synapseProject.yml [hunk 3]
```diff
@@ -12,7 +12,15 @@ post:
     security:
         - BridgeSecurity: []
     parameters:
-        - $ref: ../../index.yml#/parameters/synapseUserId
+        - name: body
+          required: true
+          in: body
+          schema:
+            type: array
+            description: A list of synapse user ids to add into new Project and Team
```
Comment (alxdarksage) [inline bridge-api/paths/studies/v3_studies_self_synapseProject.yml]: To add as administrators, right?

**Review by alxdarksage — CHANGES_REQUESTED** [review]
Looks good, just some minor clarifications needed in the comments, mostly of the admin IDs you just added.

File: bridge-api/paths/studies/v3_studies_init.yml (hunk 1 above)
Comment (zaclinsage) [reply bridge-api/paths/studies/v3_studies_init.yml]: BTW, what is operation id for?

File: bridge-api/definitions/study_and_users.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+description: |
+    The holder object for Study, a list of User and a list of admin id
```
Comment (alxdarksage) [inline bridge-api/definitions/study_and_users.yml]: admin IDs

File: bridge-api/definitions/study_and_users.yml [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+type: object
+required:
+    - adminIds
+    - study
+    - users
+properties:
+    adminIds:
+        type: array
+        description: A list of admin id.
```
Comment (alxdarksage) [inline bridge-api/definitions/study_and_users.yml]: admin IDs (also explain: administrators of what?)

File: bridge-api/paths/studies/v3_studies_init.yml [hunk 6]
```diff
@@ -0,0 +1,23 @@
+post:
+    operationId: createStudyAndUsers
+    summary: Create a new study with given study, user list; and assigning the given Synapse users as admins of the team/project
```
Comment (alxdarksage) [inline bridge-api/paths/studies/v3_studies_init.yml]: "Create a new study with administrators and users" - summary
description: Create a new study along with the access team and project in Synapse to receive data collected through Bridge. Assigns the Synapse users in the admin ID list to be administrators of the access team and project, and creates the users in the user IDs (users will receive an email confirmation from Synapse to complete account creation).

File: bridge-api/paths/studies/v3_studies_self_synapseProject.yml [hunk 7]
```diff
@@ -12,7 +12,15 @@ post:
     security:
         - BridgeSecurity: []
     parameters:
-        - $ref: ../../index.yml#/parameters/synapseUserId
+        - name: body
+          required: true
+          in: body
+          schema:
+            type: array
+            description: A list of synapse user ids to add as admins into new Project and Team
```
Comment (alxdarksage) [inline bridge-api/paths/studies/v3_studies_self_synapseProject.yml]: A list of synapse user IDs to add as admins of the created project and access team

**Review by alxdarksage — COMMENTED** [review]
Just change to what I've written, then feel free to submit.

**Pull request merged**